web/src/Web.App/Views/Home/Index.cshtml

```diff
@@ -8,50 +8,33 @@
     </div>
 </div>
 
-<div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-        <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
-            <div class="govuk-notification-banner__header">
-                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
-                    Important
-                </h2>
-            </div>
-            <div class="govuk-notification-banner__content">
-                <h3 class="govuk-notification-banner__heading">The Schools Financial Benchmark and View My Financial Insights services have moved</h3>
-                <p class="govuk-body">You can now find the tools from those services here in Financial Benchmarking and Insights Tool</p>
-            </div>
-        </div>
-    </div>
-</div>
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <p class="govuk-body">This services helps schools to view and improve their financial performance</p>
-        <p class="govuk-body">You can compare a school or trust with similar locations in England.</p>
-        <p class="govuk-body">At a school level you can:</p>
+        <p class="govuk-body">This service helps schools view and improve their spending. It is replacing the View my financial insights tool and the Schools financial benchmarking service.</p>
+        <p class="govuk-body">
+            Anyone can use it to compare schools, academies or trusts across England, though you will need 
+            <a href="https://services.signin.education.gov.uk/" class="govuk-link govuk-link--no-visited-state">a DfE Sign-in account</a> 
```

Comment:
This needs to come from configuration. As the URL is different between prod and pre-prod.